delatore/outputs/telegram/bot.py

```diff
@@ -131,7 +128,7 @@ async def start_posting(self):
             message = await self.client.get(.1)
             if message is not None:
                 data = json.loads(message)
-                if (len(data['status_list'])) > 0 and ('status_list' in data):
+                if 'status_list' in data and len(data['status_list']) > 0:
```

Comment:
`len(data['status_list']) > 0` can be simplified to `data['status_list']`

Reply:
fixed